perceval/backends/opnfv/functest.py

```diff
@@ -69,8 +68,25 @@ def fetch(self, from_date=DEFAULT_DATETIME, to_date=None):
 
         :returns: a generator of items
         """
+        if not from_date:
```

Comment:
It's easier to write:
```
from_date = datetime_to_utc(from_date) if from_date else DEFAULT_DATETIME
to_date = datetime_to_utc(to_date) if to_date else datetime_utcnow()
```

Both `DEFAULT_DATETIME` and `datetime_utcnow` give dates with UTC timezone.